README.md

```diff
@@ -10,41 +10,25 @@ UltraStar Play is a free and open source karaoke singing game for Windows, Linux
 The game uses the Unity 2D framework and mostly c# (mono/.Net) as software development language.
 
 ### 2. Minimal Game Requirements
-- Windows Vista SP1 or Android OS 4.1 or iOS 7.0 or Mac OS X 10.9, Ubuntu 12.04, SteamOS or any newer
-- DirectX 10 with Shader Model 4.0 or OpenGL ES 2.0 or any newer
-- speakers, USB microphones (or similar), big screen
-- plenty gigabytes of storage space for songs, videos and other game content - varies depending on quantity and quality
+- Windows 7 SP1+, macOS 10.12+, Ubuntu 16.04+
+- Graphics card with DX10 (shader model 4.0) capabilities.
+- iOS player requires iOS 9.0 or higher.
+- Android: OS 4.1 or later; ARMv7 CPU with NEON support or Atom CPU; OpenGL ES 2.0 or later.
+- Speakers, USB microphones (or similar)
 
 ### 3. Development Requirements
-- Windows 7 SP1+, 8, 10, only 64-bit; Mac OS X 10.9+.
-- DX10 with Shader Model 4.0+
-- plenty of free time
-- see [How to Compile the Game](https://github.com/UltraStar-Deluxe/Play/wiki/Compiling-the-game)
+- See [Unity system requirements](https://unity3d.com/unity/system-requirements)
+- Plenty of free time
 
 ### 4. Documentation
-see [the project wiki](https://github.com/UltraStar-Deluxe/Play/wiki)
+- See [the project wiki](https://github.com/UltraStar-Deluxe/Play/wiki)
 
 ### 5. How to contribute
-see [CONTRIBUTING.md](https://github.com/UltraStar-Deluxe/Play/blob/master/CONTRIBUTING.md) and also [How to Compile the Game](https://github.com/UltraStar-Deluxe/Play/wiki/Compiling-the-game)
+- See [CONTRIBUTING.md](https://github.com/UltraStar-Deluxe/Play/blob/master/CONTRIBUTING.md) and also [How to Compile the Game](https://github.com/UltraStar-Deluxe/Play/wiki/Compiling-the-game)
+- See [How to Compile the Game](https://github.com/UltraStar-Deluxe/Play/wiki/Compiling-the-game)
```

Comment:
the Compiling-the-game page is mentioned twice here

Reply:
True. I think I merged two sections which already where redundant. Feel free to change this as you please